client/state/data-layer/wpcom/activity-log/rewind/to/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_RESTORE,
+} from 'state/action-types';
+import {
+	rewindCompleteRestore,
+	rewindRestoreUpdateError,
+} from 'state/activity-log/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+const requestRestore = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'POST',
+		path: `/activity-log/${ action.siteId }/rewind/to/${ action.timestamp }`,
+		apiVersion: '1',
+	}, action ) );
+};
+
+export const receiveRestoreSuccess = ( { dispatch }, { siteId, timestamp } ) => {
+	dispatch( rewindCompleteRestore( siteId, timestamp ) );
```

Comment:
I think this API call will actually return immediately, so it is wrong to say that the restore is complete at this point. However, until we add code to poll D5890-code, we can probably leave this here?